Source/WebCore/page/Quirks.cpp

```diff
@@ -1041,6 +1041,17 @@ bool Quirks::shouldAvoidPastingImagesAsWebContent() const
 #endif
 }
 
+bool Quirks::shouldInjectYouTubeFullsceenStyles() const
```

Comment:
```suggestion
bool Quirks::shouldInjectYouTubeFullscreenStyles() const
```

Reply:
Also, I wonder if the quirk name should be generic rather than website specific. I doubt YouTube is the only affected site

Reply:
Thanks for the catch!